### PR TITLE
Updated Definitions

### PR DESCRIPTION
**Pull request by darahayes**

### CODE REVIEW

File: definitions/services.js [hunk 1]
```diff
@@ -5,11 +5,10 @@ exports.root = {
 exports.doc = {
   type: 'process',
   specific: {
-    repositoryUrl: 'https://github.com/nearform/sudc-doc.git',
-    buildScript: 'build.sh',
+    repositoryUrl: 'https://github.com/kamil-mech/sudc-doc.git',
```
Comment (mcollina) [inline definitions/services.js]: can you please restore this and all the others to the nearform organization?

**Comment by darahayes** [comment]
Links should be fixed now

File: definitions/services.js [hunk 2]
```diff
@@ -42,8 +39,7 @@ exports.real = {
 exports.web = {
   type: 'process',
   specific: {
-    repositoryUrl: 'https://github.com/nearform/sudc-web.git',
-    buildScript: 'build.sh',
+    repositoryUrl: 'https://github.com/darragh-hayes/sudc-web.git',
```
Comment (mcollina) [inline definitions/services.js]: This is still wrong.

File: definitions/services.js [hunk 3]
```diff
@@ -30,11 +28,10 @@ exports.hist = {
 exports.real = {
   type: 'process',
   specific: {
-    repositoryUrl: 'https://github.com/nearform/sudc-realtime.git',
-    buildScript: 'build.sh',
+    repositoryUrl: 'https://github.com/nearform/sudc-real.git',
```
Comment (mcollina) [inline definitions/services.js]: should this be sudc-realtime?

**Pull request merged**